internal/benchrunner/runners/rally/runner.go

```diff
@@ -263,7 +276,27 @@ func (r *runner) run() (report reporters.Reportable, err error) {
 		return nil, fmt.Errorf("can't reindex data: %w", err)
 	}
 
-	return createReport(r.options.BenchName, r.corporaFile, r.scenario, msum, rallyStats)
+	return createReport(r.options.BenchName, r.corpusFile, r.scenario, msum, rallyStats)
+}
+
+func (r *runner) installPackage(packageName, packageVersion string) error {
```

Comment:
As [commented in other thread](https://github.com/elastic/elastic-package/pull/1522#discussion_r1378669668), consider installing packages as system tests do to avoid depending on the package registry when not needded: https://github.com/elastic/elastic-package/blob/a7e19f9823f38bff7822a4bd3c61f5f61d4bb81a/internal/testrunner/runners/system/runner.go#L490

Reply:
done